Source/WTF/wtf/Compiler.h

```diff
@@ -268,16 +277,10 @@
 // written for 64-bit may fail to tail call on 32-bit.
 // It also doesn't work on ppc64le: https://github.com/llvm/llvm-project/issues/98859
 // and on Windows: https://github.com/llvm/llvm-project/issues/116568
-#if COMPILER(CLANG)
-#if __SIZEOF_POINTER__ == 8
-#if !defined(MUST_TAIL_CALL) && defined(__cplusplus) && defined(__has_cpp_attribute)
-#if __has_cpp_attribute(clang::musttail) && !defined(__powerpc__) && !defined(_WIN32)
+#if !defined(MUST_TAIL_CALL) && COMPILER(CLANG) && __SIZEOF_POINTER__ == 8 && defined(__cplusplus) && COMPILER_HAS_CPP_ATTRIBUTE(clang::musttail) && !defined(__powerpc__) && !defined(_WIN32)
```

Comment:
Not new, but seems like overkill to do both `COMPILER(CLANG)` and `COMPILER_HAS_CPP_ATTRIBUTE(clang::musttail)`.